geotrek/core/helpers.py

```diff
@@ -62,7 +62,7 @@ def deserialize(cls, serialized):
             pk = objdict.get('pk')
             kind = objdict.get('kind')
             # Point topology ?
-            if lat and lng:
+            if lat is not None and lng is not None:
```

Comment:
çà ne change rien. if lat and lng teste deja si c'est pas None et pas vide

Reply:
en fait laissera passer les chaines vides

Reply:
non c'est juste pour les lat en 0
:)